xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-sources/src/main/java/org/xwiki/notifications/sources/ParametrizedNotificationManager.java

```diff
@@ -19,11 +19,14 @@
  */
```

Comment:
I noticed only now that there is a typo in the name of that API, but it's a bit late now...